tests/bench_timers/Makefile

```diff
@@ -53,4 +51,5 @@ test-kinetis-lptmr: all
 # Reset the default goal.
 .DEFAULT_GOAL :=
 
+include ../Makefile.tests_common
```

Comment:
This one should also be before the 

```
ifeq (,$(findstring TIM_TEST_DEV,$(CFLAGS)))
```

This handling is already a bit dirty so would not touch it with this PR.